CMake/FindIBMMASS.cmake

```diff
@@ -18,17 +18,15 @@ set(MASS_FIND_LIB "libmass${CMAKE_STATIC_LIBRARY_SUFFIX}")
 set(SUFFIXES lib lib64)
 find_path(MASS_LINK_DIRECTORIES name "${MASS_FIND_LIB}" HINTS ${MASS_ROOT}
   PATH_SUFFIXES ${SUFFIXES})
-if (MASS_LINK_DIRECTORIES-NOTFOUND)
+if (NOT MASS_LINK_DIRECTORIES)
   message(FATAL_ERROR "MASS_LINK_DIRECTORIES not set. ${MASS_FIND_LIB} "
     "not found in MASS_ROOT/(${SUFFIXES})")
-endif (MASS_LINK_DIRECTORIES-NOTFOUND)
+endif (NOT MASS_LINK_DIRECTORIES)
 message("MASS_LINK_DIRECTORIES: ${MASS_LINK_DIRECTORIES}")
+
 set(MASS_LINKER_FLAGS -L${MASS_LINK_DIRECTORIES} -Wl,-rpath,${MASS_LINK_DIRECTORIES})
-if (CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL ppc64le)
```

Comment:
power8 is also ppb64le and I don't feel good to hard code p9.

Reply:
True I forgot I had not resolved this nicely.  Really your OLCF email is how this would be better resolved i.e. sym-linking the mass library "generic versions" to the correct architecture.  But since we can't count on that I will find a work around.

Reply:
I will find a proper work around for this, since I guess we can't count on computing centers or IBM to simlink the correct architecture specific library.